Mirror/Runtime/Transport/EpicOnlineTransport/Lobby/EOSLobby.cs

```diff
@@ -256,6 +256,36 @@ public class EOSLobby : MonoBehaviour {
         });
     }
 
+    public virtual void JoinLobbyByID(string lobbyID)
+    {
```

Comment:
Make sure this bracket is on the same line as the method name

Reply:
Fixed :+1: